packages/vim_runtime.rb

```diff
@@ -3,49 +3,66 @@
 class Vim_runtime < Package
   description 'Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient. (shared runtime)'
   homepage 'http://www.vim.org/'
-  version '8.2.1976'
+  @_ver = '8.2.2580'
+  version @_ver
   compatibility 'all'
-  source_url 'https://github.com/vim/vim/archive/v8.2.1976.tar.gz'
-  source_sha256 'd2d8bc28e28e9c5a63be570cdb44be39470621bb57dcbace5abbd86e15690678'
+  source_url 'https://github.com/vim/vim/archive/v8.2.2580.tar.gz'
+  source_sha256 'd0a508ca9726c8ff69bc5f5ab1ebe251c256e01e730f7b36afd03a66c89fcf79'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.1976-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.1976-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.1976-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.1976-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.2580-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.2580-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.2580-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vim_runtime-8.2.2580-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '681fbb6f06cfb300a1a1ba529bc9596ae3fc183e5ccc5e77a8bfb5869da4c281',
-     armv7l: '681fbb6f06cfb300a1a1ba529bc9596ae3fc183e5ccc5e77a8bfb5869da4c281',
-       i686: 'd6c5b6b3518c4a7856df53fac5f18af5bf6d48094bf1dc11d46018ba92cb2298',
-     x86_64: '588ce10bf392d363ce9de709f763ab79ef35a5e8e5eb67e278eec894b6fa04ed',
+  binary_sha256({
+    aarch64: '89f2375c66810417550d4ab2d2fbf90a6953e3c585b8e0baaa1240b7e0187c99',
+     armv7l: '89f2375c66810417550d4ab2d2fbf90a6953e3c585b8e0baaa1240b7e0187c99',
+       i686: '0b7505d98e974e45d621f33fcaa19f14d907422d3b04f41bfdb0d9143776d61d',
+     x86_64: 'a2ee9fcb92bc566200d21a176ea7bfec6ef7d5b3fbe3fc3c0244ed31e6168d14'
   })
 
+  def self.preflight
+    raise StandardError, 'Please remove libiconv before building.' if File.exist?("#{CREW_LIB_PREFIX}/libcharset.so")
```

Comment:
Better to use `abort`? (Like a warning, not an runtime error)

Reply:
Does this look better?
`abort('Please remove libiconv before building.') if File.exist?("#{CREW_LIB_PREFIX}/libcharset.so")`